docs/portal/applications/updating-an-application.md

```diff
@@ -1,24 +1,66 @@
-# Updating an Application
+# Quick Updating an Application
 
 The following procedure describes how to update applications from the Applications page. You can also update applications from the form used to [view application information](</docs/portal/applications/viewing-application-information.md>).
 
 1. In the left pane, click **Applications**.
-2. Find the application you want to update, then click its **Actions** menu and select **Update**. An Update Application form similar to the following appears.
+2. Click the name of the application you want to update.
+3. At the top right of the form, click the **Update** button, and then select one of the following quick update options:<ul><li> **Redistribute** allows you to add, edit, and delete distribution targets.</ul></li><ul><li>**Set container image** allows you to update your pods to a newer application version.br></ul></li><ul><li>**Upgrade resources** allows you to upgrade the spec of your container instance to maintain performance and/or save costs.</ul></li>
```

Comment:
There are some unexpected characters at the end of the second list item, ".br>".

Reply:
Fixed